sdk-actors/src/test/java/io/dapr/actors/client/ActorProxyImplTest.java

```diff
@@ -23,7 +23,7 @@
 
   @Test()
   public void constructorActorProxyTest() {
-    final DaprClient daprClient = mock(DaprClient.class);
+    final ActorClient daprClient = mock(ActorClient.class);
```

Comment:
my suggestion is let ActorClient implement DaprClient interface.

Reply:
I had this in a version but changed. The reason is that it would expose DaprClient method publicly while it is an internal interface. It would allow users to bypass the SDK and invoke the actor APIs directly without using ActorProxy.